graf3d/ftgl/CMakeLists.txt

```diff
@@ -46,4 +44,8 @@ ROOT_LINKER_LIBRARY(FTGL
     FREETYPE
 )
 
+target_include_directories(FTGL PRIVATE ${FREETYPE_INCLUDE_DIRS})
```

Comment:
Here next two lines could be more combined in single call of target_include_directories() [see below]

Reply:
@oshadura I will keep it as is, otherwise will have more problem rebasing code from #5181 
I think both variants are equivalent

Reply:
I think this PR should be 1 single commit saying: “Modernize cmake by using target_include_directories rather than include_directories”. There is no need to bloat the history and make the intent less obvious. This would require rebasing and give a chance to Oksana’s comment.

Reply:
Squash on merge directly on github is also an option.